Packs/CVE_2023_34362_-_MOVEit_SQLI/Playbooks/playbook-CVE-2023-34362_-_MOVEit_Transfer_SQL_Injection.yml

```diff
@@ -984,14 +926,6 @@ tasks:
               right:
                 value:
                   simple: file
-          - - operator: greaterThanOrEqual
-              left:
-                value:
-                  simple: DBotScore.Score
-                iscontext: true
-              right:
-                value:
-                  simple: "3"
           accessor: Indicator
```

Comment:
there is no key for file.indicator.

should be file.sha254

Reply:
Done